support/builder/config.sh

```diff
@@ -78,7 +78,6 @@ log_level = "debug"
 
 [datastore]
 password = "$PGPASSWORD"
-database = "builder_jobsrv"
```

Comment:
How does jobsrv connect if this is removed?

Reply:
it's the default in `default.toml`